script/tool/lib/src/common/package_command.dart

```diff
@@ -522,6 +533,18 @@ abstract class PackageCommand extends Command<void> {
     return packages;
   }
 
+  // Retruns true if the environment indicates that the current treeish is from
```

Comment:
Returns typo
treeish?

Reply:
Fixed, but also rewrote because I forgot to update this comment after coming up with this implementation (instead of special-casing LUCI based on environment variables).

That means it doesn't need to refer to treeish any more, but it's [a git term](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddeftree-ishatree-ishalsotreeish), which encompasses things like branches, tags, and hashes, etc.